.github/workflows/php-tests.yml

```diff
@@ -16,18 +16,20 @@ jobs:
         include:
           # WooCommerce
           - woocommerce_support_policy: L
-            woocommerce: '9.5.2'
+            woocommerce: '9.7.1'
           - woocommerce_support_policy: L-1
-            woocommerce: '9.3.4'
+            woocommerce: '9.6.2'
           - woocommerce_support_policy: L-2
-            woocommerce: '9.2.3'
+            woocommerce: '9.5.2'
           # WordPress
           - wordpress_support_policy: L
-            wordpress: '6.7'
+            wordpress: '6.7.2'
           - wordpress_support_policy: L-1
-            wordpress: '6.6'
+            wordpress: '6.6.2'
+          # WooCommerce 9.5.0+ requires WordPress 6.6+
+          # (we'll keep two versions from the 6.6 branch until Apr when WP 6.8 is released)
           - wordpress_support_policy: L-2
-            wordpress: '6.5'
+            wordpress: '6.6'
```

Comment:
WooCommerce dropped support for `WordPress 6.5` in version `9.5` (our L-2), for consistency we keep 3 versions; with both the newest and the oldest versions from the 6.6 branch.